gallery/plot_repurposing_annotations.py

```diff
@@ -0,0 +1,75 @@
+"""
+=======================
+Repurposing annotations
+=======================
+
+The following example illustrates the operations available in :ref:`the torchvision.ops module <ops>` for repurposing
```

Comment:
After some debugging I found out the reason for `build_docs` CI failure. The problem is `torchvision.ops` does not have a nice index on right side (basically a html link to #ops like transforms has). This causes CI failure.

We need to remove the ref, and it will work fine.
I tried running it locally. I could build the gallery example. It looks nice.

```suggestion
The following example illustrates the operations available in the torchvision.ops module for repurposing
```

Reply:
Nice! I appreciate the debugging.